com.unity.cinemachine/Runtime/Behaviours/CinemachineClearShot.cs

```diff
@@ -85,7 +85,7 @@ protected internal override void LegacyUpgrade(int streamedVersion)
             {
                 DefaultTarget = new DefaultTargetSettings 
                 { 
-                    Enabled = m_LegacyLookAt != null || m_LegacyFollow != null,
```

Comment:
Why this change?

Reply:
Fixes: [CMCL-1157](https://jira.unity3d.com/browse/CMCL-1157) - we may need to come up with something else. LegacyUpgrade cannot check for null, because that is only allowed on the main thread. However, OnAfterDeserialize is not guaranteed to run on the main thread. When it does not run on main thread we get an error that stops LegacyUpgrade.
(Error message about EnsureRunningOnMainThread)